src/server/examples_test.go

```diff
@@ -0,0 +1,81 @@
+package server
+
+import (
+	"bytes"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/pachyderm/pachyderm/src/client"
+	pfsclient "github.com/pachyderm/pachyderm/src/client/pfs"
+	"github.com/pachyderm/pachyderm/src/client/pkg/require"
+)
+
+func TestTensorFlow(t *testing.T) {
+	if testing.Short() {
+		t.Skip("Skipping integration tests in short mode")
+	}
+
+	cwd, err := os.Getwd()
+	require.NoError(t, err)
+	exampleDir := filepath.Join(cwd, "../../examples/tensor_flow")
+	cmd := exec.Command("make", "all")
+	cmd.Dir = exampleDir
+	raw, err := cmd.CombinedOutput()
+	require.NoError(t, err)
+
+	cmd = exec.Command("pachctl", "list-commit", "GoT_scripts")
```

Comment:
Hmmm ... this seems a bit overkill on second consideration. We have good coverage for list-commit already ... so there's no reason to deal w stdout parsing for list-commit. I'll change this to use the API directly